scalardb-sample/README.md

```diff
@@ -113,26 +115,36 @@ scalar.db.username=cassandra
 scalar.db.password=cassandra
 ```
 
-Since you use Cassandra in this sample application as mentioned above, you need to configure Cassandra settings in the configuration.
+Since this sample application uses Cassandra, as shown above, you need to configure your settings for Cassandra in this configuration.
+
+## Setup Cassandra
 
-## Set up
+To start Cassandra, you need to run the following `docker-compose` command:
 
-You need to run the following `docker-compose` command:
 ```shell
 $ docker-compose up -d
 ```
 
-This command starts Cassandra and loads the schema.
-Please note that you need to wait around more than one minute for the containers to be fully started.
+Please note that starting the containers may take more than one minute.
 
-### Initial data
+## Load schema
+
+You then need to apply the schema with the following command.
+Please download the schema tool `scalardb-schema-loader-<version>.jar` that can be found in [releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB.
+
+```shell
+$ java -jar scalardb-schema-loader-<version>.jar --config database.properties --schema-file schema.json --coordinator
+```
```

Comment:
The most major change is here.

Previously, we loaded the schema using the docker-compose command. However, I've modified the process to download the schema loader jar and load the schema using that instead.

Reply:
What is the motivation for this change?
Loading the schema through docker-compose simplifies things for the user and reduces the barrier to entry to try out ScalarDB. 
If the motivation for the change is to show how to use the schema loader,  we can offer two alternatives in the sample: load the schema using the schema loader docker-compose image or via the downloaded jar.

Reply:
Sorry for the lack of the explanation.

I made this change because, in practice, we basically use the Schema Loader tool in production, rather than the schema loader docker image. We assume that users basically run ScalarDB components in Kubernetes environments. In such environments, the Schema Loader tool is easier to use than the schema loader docker image.

Reply:
For this, I think it should probably be fine.
But I'm wondering how we should create a database schema in a production environment.

ScalarDL now uses the schema-loader container and the corresponding helm chart to load the schema.
Should ScalarDB also do that way? or should it just use `java -jar`?
Java jar requires a Java runtime, so it will probably be executed from your local.
Then, I feel using `java -jar` from your local to a production cluster could probably be restricted for some security reasons in production environments in enterprises.
So, it might be better to load a schema through a container that contains the required jar file.

@brfrn169 @kota2and3kan

What do you think?

Reply:
If you use the schema-loader container in Kubernetes, you need to create ConfigMaps for the config file and the schema JSON file, and create a Pod for the schema-loader container as follows:

```shell
# Create a ConfigMap for database.properties
kubectl create cm database-properties --from-file=database.properties

# Create a ConfigMap for schema.json
kubectl create cm schema-json --from-file=schema.json

# Create a Pod for the schema-loader container
kubectl apply -f scalardb-schema-loader.yaml
```

scalardb-schema-loader.yaml
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: scalardb-schema-loader
spec:
  volumes:
  - name: database-properties
    configMap:
      name: database-properties
  - name: schema-json
    configMap:
      name: schema-json
  containers:
  - name: scalardb-schema-loader
    image: ghcr.io/scalar-labs/scalardb-schema-loader:3.9.0
    args: ["--config", "/database.properties", "--schema-file", "/schema.json", "--coordinator"]
    volumeMounts:
    - name: database-properties
      mountPath: /database.properties
      subPath: database.properties
    - name: schema-json
      mountPath: /schema.json
      subPath: schema.json
  restartPolicy: Never
  terminationGracePeriodSeconds: 0
```

Instead of this approach, I think that using the Schema Loader tool (`java -jar`) is simpler and more user-friendly. Of cause, in this case, the Schema Loader tool must be able to connect to the database (or ScalarDB Server). This could potentially lead to security issues, but I think this risk is equivalent to that when using the schema-loader container.

@feeblefakie What do you think?

Reply:
(Sorry, it's a sample repo, so it's a bit off-topic.)

I totally agree with you that `java -jar` is way simpler, so it is better to use it for development.
I'm just saying that it might not be for production because you don't usually have direct access to a production cluster from a local computer (that has Java runtime) for security reasons in enterprises.
We could probably run `java -jar` by doing multi-level port-forwarding through a bastion server, but it might be tricky.

I reacted to the following message and just realized that we should be on the same page for this, but it's not about the PR since the sample repo is for development, so let's discuss it offline.
> we basically use the Schema Loader tool in production

Reply:
@feeblefakie @brfrn169  
Sorry for the late response. From the technical perspective, I think we can create a helm chart of ScalarDB Schema Loader like ScalarDL Schema Loader. For example, in this chart, users can set both `database.properties` and `schema.json` in one custom values file as follows.

```yaml
scalardbSchemaLoader:

  databaseProperties: |
    scalar.db.storage=...
    scalar.db.contact_points=...
    scalar.db.username=...
    scalar.db.password=...

  schemaJson: |
    {
        "s0.t0": {
            "transaction": true,
            "partition-key": [
                "c1"
            ],
            "clustering-key": [
                "c2"
            ],
            "columns": {
                "c1": "INT",
                "c2": "INT",
                "c3": "TEXT"
            }
        }
    }
```

And uses can deploy ScalarDB Schema Loader as follows.
```console
$ helm install <RELEASE_NAME> scalar-labs/scalardb-schema-loader -f <CUSTOM_VALUES_FILE>
```

Let's discuss it offline!